### PR TITLE
Update sanityCheck doc tests in samples/

### PR DESCRIPTION
**Pull request by mlopatkin**
Do not run `tasks` in the named tests, these are already covered with sanityCheck.

Part of #21027

### CODE REVIEW

**Review by nathan-contino — APPROVED** [review]
LGTM

File: subprojects/docs/src/samples/build-organization/composite-builds/basic/tests/sanityCheck.sample.conf [hunk 1]
```diff
@@ -1,12 +1,12 @@
 commands: [{
     execution-subdirectory: my-app
     executable: gradle
-    args: help -q
+    args: tasks -q
```
Comment (6hundreds) [inline subprojects/docs/src/samples/build-organization/composite-builds/basic/tests/sanityCheck.sample.conf:4]: @mlopatkin Btw, does this change is same in terms of test coverage?
I mean that in both cases we covering only configuration phase.

**Review by 6hundreds — APPROVED** [review]
LGTM

File: subprojects/docs/src/samples/build-organization/composite-builds/basic/tests/sanityCheck.sample.conf (hunk 1 above)
Comment (mlopatkin) [reply subprojects/docs/src/samples/build-organization/composite-builds/basic/tests/sanityCheck.sample.conf:4]: No, it isn't the same. The `help` only configures the `help` task (if the build author follows the recommendations and uses the lazy API, like `register`). This means that configuration blocks are not executed, and may allow errors to creep in, esp. in the case of Groovy scripts. The `tasks` task eagerly configure all other tasks, avoiding this problem. The updated plugin uses `tasks` as a sanity check target, but it cannot infer the structure of a composite build or a build that resides in a subfolder. For such snippets we have to override the generated sanity check test, and this is what I'm doing in this PR.

**Review by mlopatkin — COMMENTED** [review]
> @mlopatkin You wanted to mention `help` instead of `tasks` in PR description? Not a nitpicking, just wanted to be confident that I'm not missing something

Well, in the end, we don't want to run neither `help`, nor `tasks` in named tests. I guess my choice of words is confusing.

**Comment by mlopatkin** [comment]
@bot-gradle test and merge

File: subprojects/docs/src/samples/build-organization/composite-builds/basic/tests/sanityCheck.sample.conf (hunk 1 above)
Comment (6hundreds) [reply subprojects/docs/src/samples/build-organization/composite-builds/basic/tests/sanityCheck.sample.conf:4]: Got it, thank you!

**Review by bamboo — COMMENTED** [review]
👍

**Pull request merged**